starwars/src/components/CardComponent/CardContainer.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import CardContent from './CardContent.js';
+
+import './Card.css';
+
+export default function CardContainer(props) {
+    return (
+      props.starwarsChars.map(char => <CardContent

```

Comment:
I look at this code and i can't stop thinking that it could be improved to be more DRY. You could just pass the char as character object to CardContent and then call on that passed object in separate component. It will be less repetitive and will look better.